purge-cluster.yml

```diff
@@ -11,10 +11,13 @@
 
   become: yes
 
-  roles:
-    - ceph-common
-
   vars:
+    osd_group_name: osds
+    mon_group_name: mons
+    rgw_group_name: rgws
+    mds_group_name: mdss
+    process_success: 0  # process exit status for success
```

Comment:
I updated the pull request to remove it.

----- Original Message -----
> From: "Leseb" <notifications@github.com>
> To: "ceph/ceph-ansible" <ceph-ansible@noreply.github.com>
> Cc: "Ben England" <bengland@redhat.com>
> Sent: Wednesday, March 23, 2016 5:16:05 AM
> Subject: Re: [ceph-ansible] make purge-cluster.yml both more reliable and faster (#644)
> 
> >    vars:
> > +    osd_group_name: osds
> > +    mon_group_name: mons
> > +    rgw_group_name: rgws
> > +    mds_group_name: mdss
> > +    process_success: 0  # process exit status for success
> 
> Looks like this is never used. What's the purpose of this variable?
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ceph/ceph-ansible/pull/644/files/690fd2c70d73e0166e79dd1d217fff95c4f3bec2#r57128205